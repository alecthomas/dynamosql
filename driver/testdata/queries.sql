SELECT * FROM gamescores WHERE UserId = "101"
SELECT * FROM gamescores WHERE UserId = "101" AND TopScore > 30
SELECT * FROM gamescores WHERE UserId = "103" AND GameTitle BETWEEN "Galaxy" AND "Meteor" AND TopScore > 1000
SELECT GameTitle, TopScore FROM gamescores WHERE UserId = "101"
SELECT title, year FROM movies WHERE title = "World War Z"
-- info.rating selects from {"info": {"rating": 7.1}}
SELECT title, info.rating FROM movies WHERE title = "World War Z"
-- `info.rating` selects the field "info.rating" which does not exist
SELECT title, `info.rating` FROM movies WHERE title = "World War Z"
