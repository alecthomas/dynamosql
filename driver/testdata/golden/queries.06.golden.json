{
  "Query": "SELECT title, `info.rating` FROM movies WHERE title = \"World War Z\"",
  "Results": [
    "title,info.rating",
    "World War Z,NULL"
  ]
}
